src/app/components/harvest/screens/uploading/batch-uploading.component.html

```diff
@@ -102,64 +119,74 @@ <h3>Uploading Files</h3>
 
         <ng-container>
           <p>
-            We have created some folders (using the site name and id for the
-            folder name) for you to put your files in already, however you are
-            not required to use them. Here is an example of one:
-            </p>
+            We have created some default folders for each of your
+            {{ hideProjects ? "point" : "site" }}s. They use the
+            <i>{{ hideProjects ? "point" : "site" }} name</i> along with a
```

Comment:
`hideProjects` is not the correct switch to use here.

Ecosounds has `hideProjects` set to `false` and it has projects which have sites which have points.

You have to query to see if the current project has any `regions`. It's probably easier for the moment to do `points/sites` in the text